tds/src/main/java/thredds/server/wfs/WFSXMLGeneral.java

```diff
@@ -0,0 +1,26 @@
+package thredds.server.wfs;
+
+/**
+ * General XML writing utilities.
+ * 
+ * @author wchen@usgs.gov
+ *
+ */
+public class WFSXMLGeneral {

```

Comment:
Few classes are named "General"s. To help subsequent programmers, could you pick a more widely-understood term for the name? Would "Util" or "Utility" fit? Would "Helper"?

Reply:
Sure rename to "Helper"